interfaces/automated-testing/scd/design/strategic-coordination-interoperability/nominal-test.plantuml

```diff
@@ -0,0 +1,28 @@
+@startuml
+center header Strategic Coordination Interoperability
+title Nominal test
+
+participant "Test Driver" as TD
+participant "USS1 under test" as USS1
+participant "USS2 under test" as USS2
+participant "DSS"
+
+== TD injects via USS1 non-conflicting Operational Intent 1 ==
```

Comment:
This is a architectural question we (you) need to decide, do we inject Operational Intents or do you want to send them Volume4Ds and then the USS creates Operational Intents, I know that it is a small / minor thing but the reason I ask is because [OperationalIntent contains ](http://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/astm-utm/Protocol/master/utm.yaml#operation/createOperationalIntentReference)details that the test driver / simulator may or may not know. My suggestion is that we submit Volume4D and let the USS handle creation of operational intents etc. Otherwise we may have to submit "partial" operational intents.

Reply:
The test driver is acting as a simulated user attempting to utilize the system, so I don't think there should be anything about the flight it doesn't know (it is making up everything about the flight).  I don't think giving the USS just the Volume4D is sufficient for the USS to create an operational intent on behalf of the user (qualifier) -- the USS doesn't know what the priority of the user's flight is, for instance, nor do they know whether this is a flight being planned for the future (targeting Accepted state) or a flight about to take off (targeting Activated state) or a rogue flight (targeting Contingent state), nor do they know if this is a new flight or an update to an existing flight.  When the qualifier injects an instruction to create an operational intent, I think all of this information needs to be supplied one way or another.

Reply:
Thanks! That's great I will start working on the changes to the simulator and the API to accommodate / generate Operational Intents (instead of Volume4Ds).